api/scripts/reprocess_transfers.py

```diff
@@ -10,52 +10,57 @@
+    tokenQueue = SqsMessageQueue(region='us-east-1', accessKeyId=os.environ['AWS_KEY'], accessKeySecret=os.environ['AWS_SECRET'], queueUrl='https://sqs.us-east-1.amazonaws.com/113848722427/FemiKiBa')
     slackClient = SlackClient(webhookUrl=os.environ['SLACK_WEBHOOK_URL'], requester=requester, defaultSender='worker', defaultChannel='notd-notifications')
     awsRequester = AwsRequester(accessKeyId=os.environ['AWS_KEY'], accessKeySecret=os.environ['AWS_SECRET'])
     ethClient = RestEthClient(url='https://nd-foldvvlb25awde7kbqfvpgvrrm.ethereum.managedblockchain.eu-west-1.amazonaws.com', requester=awsRequester)
-    blockProcessor = BlockProcessor(ethClient=ethClient)
     notdManager = NotdManager(blockProcessor=None, saver=saver, retriever=retriever, workQueue=None, tokenManager=None, requester=requester, revueApiKey=None)
 
     await database.connect()
-    await slackClient.post(text=f'reprocess_transfers → 🚧 started: {startBlockNumber}-{endBlockNumber}')
-    try:
-        currentBlockNumber = startBlockNumber
-        while currentBlockNumber < endBlockNumber:
-            logging.info(f'Working on {currentBlockNumber}')
-            try:
-                retrievedTokenTransfers = await blockProcessor.process_block(blockNumber=currentBlockNumber)
-            except Exception as exception:
-                logging.info(f'Got exception whilst getting blocks: {str(exception)}. Will retry in 10 secs.')
-                time.sleep(60)
-                currentBlockNumber -= 1
-                continue
-            await notdManager._save_block_transfers(blockNumber=currentBlockNumber, retrievedTokenTransfers=retrievedTokenTransfers)
-            currentBlockNumber = currentBlockNumber + 1
-        await slackClient.post(text=f'reprocess_transfers → ✅ completed : {startBlockNumber}-{endBlockNumber}')
-    except Exception as exception:
-        await slackClient.post(text=f'reprocess_transfers → ❌ error: {startBlockNumber}-{endBlockNumber}\n```{str(exception)}```')
-        raise exception
-    finally:
-        await database.disconnect()
-        await requester.close_connections()
-        await awsRequester.close_connections()
+    await workQueue.connect()
+    await tokenQueue.connect()
+    currentBlockNumber = startBlockNumber
+    while currentBlockNumber < endBlockNumber:
+        start = currentBlockNumber
+        end = min(currentBlockNumber + batchSize, endBlockNumber)
+        logging.info(f'Working on {start} to {end}...')
+        async with saver.create_transaction() as connection:
+            query = TokenTransfersTable.select() \
+                        .with_only_columns([TokenTransfersTable.c.blockNumber])\
+                        .filter(TokenTransfersTable.c.blockNumber >= start) \
+                        .filter(TokenTransfersTable.c.blockNumber < end) \
+                        .where(TokenTransfersTable.c.contractAddress ==  None)\
+                        .group_by(TokenTransfersTable.c.blockNumber)
```

Comment:
```suggestion
            query = (
                TokenTransfersTable.select()
                        .with_only_columns([TokenTransfersTable.c.blockNumber])
                    .filter(TokenTransfersTable.c.blockNumber >= start)
                    .filter(TokenTransfersTable.c.blockNumber < end)
                    .where(TokenTransfersTable.c.contractAddress == None)
                    .group_by(TokenTransfersTable.c.blockNumber)
            )
```

i think this should work